{"name": "test_assertions", "status": "broken", "statusDetails": {"message": "AttributeError: 'async_generator' object has no attribute 'goto'", "trace": "page = <async_generator object page at 0x00000198073FD630>\nenv_vars = {'password': 'User@123', 'url': 'https://worldamenities.com/', 'username': 'user123@gmail.com'}\n\n    @pytest.mark.smoke\n    @pytest.mark.asyncio\n    async def test_assertions(page, env_vars):\n        url = env_vars['url']\n    \n        login_page = LoginPage(page)\n>       await login_page.Launching(url)\n\ntests\\test_WAWorld.py:57: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.LoginPage.LoginPage object at 0x00000198073E20F0>, WAurl = 'https://worldamenities.com/'\n\n    async def Launching(self, WAurl: str):\n>       await self.page.goto(WAurl)\nE       AttributeError: 'async_generator' object has no attribute 'goto'\n\npages\\LoginPage.py:29: AttributeError"}, "start": 1720145383635, "stop": 1720145383635, "uuid": "ae352690-8837-4bd0-9fa7-e0fc4de164cb", "historyId": "6d170851127fd59c90368cc8fc6a73af", "testCaseId": "6d170851127fd59c90368cc8fc6a73af", "fullName": "tests.test_WAWorld#test_assertions", "labels": [{"name": "tag", "value": "asyncio"}, {"name": "tag", "value": "smoke"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_WAWorld"}, {"name": "host", "value": "Feuji-V005L-R"}, {"name": "thread", "value": "17084-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_WAWorld"}]}